### ISSUE TITLE
To Do

### ISSUE THREAD

**Issue by sallamander** [sallamander/neural-networks-intro#5]
**Shallow Nets Chapter** 
- [x] Introduce error term in 2nd notebook, and keep it as a linear regression type problem. 
- [x] In the 2nd notebook, introduce `z` terms as the sum of inputs from a given node in a given layer. 
- [x] Fix forward propagation graph in 2nd notebook 
- [x] Introduce terminology of `shallow` in 1st or 2nd notebook (whenever you talk about **hidden layers**)
- [x] Standardize order of weights/biases across notebooks
- [x] Fix any mentions of `next notebook` in this chapter (e.g. see `keras` notebook).
- [x] Create appendix for non vector/weight version of calculations for the shallow net

**Conceptual**
- [x] Introduce terminology of connections in earlier chapters (e.g. the lines connecting nodes)
- [x] Explain use of _sigmoid activation_ terminology and change in relevant places
- [x] Consider not showing the Gradient Descent procedure in the `a` notebooks, and just showing forward and backward propagation (can link to _forward_ and _backward_ prop. in subsequent notebooks)
- [x] Discuss _weighted sum of the inputs_ as early as possible
- [x] Make note of why we're looking at errors in earlier notebooks (as opposed to plotting the function that the network learns)
- [x] In `tensorflow` notebooks, explain that we're returning the errors in training in the **code** (this is in the very last cell, and it is ambiguous where you're referring to when you say above)
- [x] Get rid of mention of returning simple values when `tf.Variable` objects are passed into session.run (2nd to last(ish) paragraph of `tensorflow` notebooks)
- [x] Fix description of keys of `feed_dict` argument in `tensorflow` notebooks - it shouldn't say 'variable names' because they are actual variable references (and not strings)

**Formatting**
- [x] Link [Neural Networks and Deep Learning](http://neuralnetworksanddeeplearning.com/index.html) somewhere. The last chapter when showing connectivity might be a good chapter to add it in to. 
- [x] Change `predicted outcomes` to `predicted values`
- [x] Fix all `reshape` commands to use `-1` instead of `len`
- [x] Remove mention of 'if not the same code' in `keras` notebooks when talking about `theano` or `tensorflow` running under the hood
